specification/dimensions/servicecategory.md

```diff
@@ -0,0 +1,54 @@
+# Service Category
+
+The Service Category is the highest-level classification of a service based on the core function of the service. Each service should have one and only one category that best aligns to its architectural design. The Service Category is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.
+
+The ServiceCategory column MUST be present and MUST NOT be null or empty. This column is restricted and MUST be one of the allowed values.
```

Comment:
What is the definition of "This column is restricted"?

Reply:
This means it's normalized. We used this for ChargeType. Thoughts on a clearer way to convey that? Happy to change them both.